src/server/project.ts

```diff
@@ -1627,6 +1627,7 @@ export abstract class Project implements LanguageServiceHost, ModuleResolutionHo
     }
 
     protected removeExistingTypings(include: string[]): string[] {
+        if (!include.length) return include;

```

Comment:
Should this return a copy? Or is it fine to share?

Reply:
Not needed since the array is just used to set the value as new include option in type aquisition. 
I am wondering if i should make this private .. not idea why this was protected.

Reply:
Could it just be `readonly string[]` at that point? (But, fine as-is)

Reply:
it wont match the TypeAcquisitions input property and thats why i left it like that.